<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDM</class>
 <widget class="QMainWindow" name="PDM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vital Health</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="heart_rate">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>99</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hydration Index:</string>
    </property>
   </widget>
   <widget class="QLabel" name="gsr">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>300</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SpO2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="spo2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>80</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>99</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>300</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>300</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>97.8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heart Rate:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="endBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>430</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>End Training</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
